rnd/autogpt_builder/.env.example

```diff
@@ -1,4 +1,5 @@
-AGPT_SERVER_URL=http://localhost:8000/api
+NEXT_PUBLIC_AGPT_SERVER_URL=http://localhost:8000/api
+NEXT_PUBLIC_AGPT_MARKETPLACE_URL=http://localhost:8001/market
```

Comment:
Just checking these are the defaults in the code as well? The user currently doesn’t have to configure the frontend at all so if these aren’t set, I don’t want it to crash